src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,82 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+};
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const amountOfPages = getNumbers(1, Math.ceil(total / perPage));
```

Comment:
Consider renaming `amountOfPages` to `numberOfPages` or `pageCount` to better convey the meaning that this variable represents a count rather than a collection.